main.py

```diff
@@ -0,0 +1,31 @@
+from fastapi import FastAPI
+from pymongo.cursor import Cursor #tools for iterating in MongoDB
+from bson.objectid import ObjectId
+from app.models import Coffee
+from app.database import coffees_collection
+
+
+app= FastAPI()
+ 
+@app.get("/coffees/", response_model=list[Coffee])
+def get_coffees():
+    coffees: Cursor = coffees_collection.find()
+    coffees_models=[]
+    for coffee in coffees: 
+        coffee['id']= str(coffee['_id'])
+        coffees_models.append(coffee)
+    return coffees_models
+
+@app.post("/coffees", response_model=Coffee )

```

Comment:
you're missing trailing slash here
```suggestion
@app.post("/coffees/", response_model=Coffee)

```